Unified/closor.py

```diff
@@ -499,7 +499,8 @@ def close(self):
                         ## make the specific relval rules and the replicas
                         ## figure the destination(s) out
                         destinations = set()
-                        if tier != "RECO" and tier != "ALCARECO":
+                        tiers_to_skip = UC.get("tiers_to_skip_relval")
```

Comment:
I think this relval logic could be replaced by something like:
```
if tier in UC.get("tiers_to_skip_relval"):
    print "tier: ",tier," is blacklisted, so skipping dataset placement for: ",out
    continue
```

Reply:
@amaltaro Initially I was thinking the same, but then I noticed there are are checks that are happening for the current workflow outside the `if Relval` condition, and if we jump out of the cycle here we would skip them too. (I do not know how needed those are, though)

Reply:
Todor, I went back to the code and besides the "no stats" bit, there is nothing else to be executed inside that loop. Unless I missed something, it would be fine to move to the next item in `for out in outputs:` if the condition I mentioned above is met.